frontend/src/components/assessment-creation/StatusTag.tsx

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import { Tag } from "@chakra-ui/react";
+
+type StatusProperty = "Draft" | "Published" | "Archived" | "Deleted";
+interface Status {
+  status: StatusProperty;
+}
+
+const StatusTag = ({ status }: Status): React.ReactElement => {
+  let color;
+  let backgroundColor;
```

Comment:
Color's for other status cases weren't specified, using placeholder values for now but they can easily be changed